packages/core/test/actions.test.ts

```diff
@@ -3753,51 +3753,6 @@ describe('cancel', () => {
     await sleep(10);
     expect(spy.mock.calls.length).toBe(0);
   });
-
-  it('should not be able to cancel a just scheduled non-delayed event to a just invoked child', async () => {
```

Comment:
I don't understand why this one was removed. What's wrong with this test? This specifically tests that an invalid usage of `cancel` does nothing. It's not like this is testing that `cancel` can cancel a non-delayed `send` - it's testing the opposite. A "negative" test case like this still serves a purpose.

Reply:
The `cancel(…)` action is only for delayed events: 

> The <cancel> element is used to cancel a delayed <send> event. […]
>
> https://www.w3.org/TR/scxml/#cancel

Reply:
Sorry, we can add this one back in. I misinterpreted it.